cmd/res/crtres.go

```diff
@@ -44,7 +45,7 @@ func init() {
 		"", "Name of the resource file")
 	CreateCmd.Flags().StringVarP(&resType, "type", "p",
 		"", "Resource type; Valid types include java, js, jsc, properties, py, wsdl, xsd, or xsl.")
-	CreateCmd.Flags().StringVarP(&resPath, "respath", "r",
+	CreateCmd.Flags().StringVarP(&resPath, "respath", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `respath` is breaking change right?